stacks/apps/exchange.yml

```diff
@@ -1391,6 +1220,7 @@ Resources:
         - { Key: prx:ops:environment, Value: !Ref EnvironmentType }
         - { Key: prx:dev:application, Value: Exchange }
       VisibilityTimeout: 5400 # 1.5 hour
+  # TODO: remove
```

Comment:
I guess this TODO is no longer valid either - I think I assumed we wouldn't need the queue at all anymore.  But if i understand this, the `process_jobs` will _add_ jobs to the say_when queue, to be processed by the other/real workers.  Right?

Reply:
yes! which is why we still need el queue de hablar donde